azure-pipelines.yml

```diff
@@ -18,6 +18,12 @@ steps:
 - script: npm ci
   displayName: 'Install dependencies'
 
+- script: npm run lint:fix
+  displayName: 'linting'
+
+- script: npm test -- --ci --watchAll=false --passWithNoTests
+  displayName: 'testing'
```

Comment:
maybe "Run tests"? up to you

Reply:
Well, for me **jobs** are like actions, so It is more readable writing the verb in continuous way, but I could change it if there are some conventions out there.